src/bitmessageqt/__init__.py

```diff
@@ -1856,22 +2059,40 @@ def addRow (address, label, type):
         self.ui.lineEditTo.completer().model().setStringList(completerList)
 
     def rerenderSubscriptions(self):
+        """TBC"""
+
         self.rerenderTabTreeSubscriptions()
 
     def click_pushButtonTTL(self):
+        """TBC"""
+
         QtGui.QMessageBox.information(self, 'Time To Live', _translate(
-            "MainWindow", """The TTL, or Time-To-Live is the length of time that the network will hold the message.
- The recipient must get it during this time. If your Bitmessage client does not hear an acknowledgement, it
- will resend the message automatically. The longer the Time-To-Live, the
- more work your computer must do to send the message. A Time-To-Live of four or five days is often appropriate."""), QtGui.QMessageBox.Ok)
+            "MainWindow",
+            ("The TTL, or Time-To-Live is the length of time that the network will hold the message."
+             "The recipient must get it during this time. If your Bitmessage client does not hear an "
+             "acknowledgement, it will resend the message automatically. The longer the Time-To-Live, "
+             "the more work your computer must do to send the message. A Time-To-Live of four or five "
+             "days is often appropriate.")), QtGui.QMessageBox.Ok)
 
     def click_pushButtonClear(self):
+        """TBC"""
+
         self.ui.lineEditSubject.setText("")
         self.ui.lineEditTo.setText("")
         self.ui.textEditMessage.setText("")
         self.ui.comboBoxSendFrom.setCurrentIndex(0)
 
     def click_pushButtonSend(self):
+        """
+        TBC
+
+        The whole network message must fit in 2^18 bytes.
+        Let's assume 500 bytes of overhead. If someone wants to get that
+        too an exact number you are welcome to but I think that it would
+        be a better use of time to support message continuation so that
+        users can send messages of any length.
+        """
```

Comment:
This not looks like the method docstring.

Reply:
You're right, wrong docs are worse than no docs. I've added a reference to the docstring to target that paragraph.

Reply:
This was just a comment mistakenly turned into docstring.